client/src/Components/Home/Home.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import {
+  GET_BANNERS_API,
+  GET_CATEGORIES_API,
+  PRIMARY_SERVER,
+} from "../../Constants/ServerUrl";
+import Categories from "../Categories/Categories";
+
+const Home = () => {
+  const [banners, setBanners] = useState([]);
+  const [categories, setCategories] = useState([]);
+
+  useEffect(() => {
+    getBanners();
+    getCategories();
+  }, []);
+
+  const getBanners = async () => {
+    debugger;
+    try {
+      const result = await axios.get(PRIMARY_SERVER + GET_BANNERS_API);
+      debugger;
```

Comment:
extract api logic to another file

Reply:
created separate file and pushed the changes